packages/agents-realtime/src/realtimeAgent.ts

```diff
@@ -9,7 +9,7 @@ import { RealtimeContextData } from './realtimeSession';
 
 export type RealtimeAgentConfiguration<TContext = UnknownContext> = Partial<
   Omit<
-    AgentConfiguration<TContext, TextOutput>,
+    AgentConfiguration<RealtimeContextData<TContext>, TextOutput>,
```

Comment:
wrapping with RealtimeContextData just appends `history` property as optional, so this is safe for both tsc and runtime.